remediation/fence-agents-remediation/limitations.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Limitations
+parent: Fence Agents Remediation
+grand_parent: Remediation
+nav_order: 3
+---
+
+## Limitations
+
+1. FAR is limited to running fence agents with only one [action](https://github.com/ClusterLabs/fence-agents/blob/main/doc/FenceAgentAPI.md#agent-operations-and-return-values), reboot.
+2. FAR remediation time can be long when the FAR pod resides in the unhealthy node since the FAR pod needs to be rescheduled on a new node, and that could already be tainted (with NoExecute), thus it would take more time.
```

Comment:
>how does the taint slow down rescheduling of the FAR pod?
It is not the taint by itself. The remediation time is just slower when the FAR pod is installed on the unhealthy node because then it would need to be rescheduled on a different node. The pod would be evicted by remediation if the taint hasn't evicted it already.

Reply:
> needs to be rescheduled on a new node, and that could already be tainted

- this is confusing, it sounds like the new node is tainted
- what taint actually?
- this is too detailed IMHO
- maybe just stop after `on a new node`?

> The pod would be evicted by remediation

which remediation?

Reply:
>which remediation?

I meant that the resource deletion. As it would accelerate the rescheduling of the FAR pod if the taint didn't evict the pod already. 

>this is confusing, it sounds like the new node is tainted

I see your point. Thanks for mentioning that.

>what taint actually?

ATM there is only one taint in FAR context. FAR NoExecute taint.

>this is too detailed IMHO
maybe just stop after on a new node?

SGTM